decoder/incident.go

```diff
@@ -66,7 +80,15 @@ func hasChangesIncident(old *Incident, new *Incident) bool {
 		old.DisplayType != new.DisplayType ||
 		old.Style != new.Style ||
 		old.Character != new.Character ||
-		old.Updated != new.Updated
+		old.Confirmed != new.Confirmed ||
```

Comment:
We should probably prevent any change if `Confirmed` is true. This way, we prevent a detected Giovanni/Decoy to be reverted to a regular Giovanni.

Reply:
The fix should be in updateFromPokestopIncidentDisplay -- to not actually make the change from the GMO if it is previously confirmed (or whatever the exact rule should be)

Reply:
Resolved in latest.